README.md

```diff
@@ -375,48 +377,47 @@ issuing stock is how companies often choose to do this.
   paper that prove you own the stock.
-- **Where is the stock certificate?**
-  Sometimes you have stock but don’t have the physical certificate, as it may be held for
+- Sometimes you have stock but don’t have the physical certificate, as it may be held for
   you at a law office.
-  Some companies now manage their ownership through online services called **ownership
+- Some companies now manage their ownership through online services called **ownership
   management platforms**, such as [Carta](https://carta.com/). If the company you work for uses
   an ownership management platform, you will be able to view your stock certificates and
   stock values online.
-  Younger companies may also choose to keep their stock **uncertificated**, which means your
+- Younger companies may also choose to keep their stock **uncertificated**, which means your
   sole evidence of ownership is your contracts with the company, and your spot on the
-  company’s capitalization table, without having a separate certificate for it.
-- 🄳 The [**outstanding shares**](http://www.investopedia.com/terms/o/outstandingshares.asp) are
-  the total number of shares at a given point, reflecting how many shares are currently held
-  by all shareholders.
+  company’s [capitalization table](https://en.wikipedia.org/wiki/Capitalization_table), without
+  having a separate certificate for it.
+- 🄳 [**Outstanding shares**](http://www.investopedia.com/terms/o/outstandingshares.asp) refer to
+  the total number of shares held by by all shareholders.
   This number starts at an essentially arbitrary value (such as 10 million) when the company
   is created, and thereafter will increase as new shares are added (issued) and granted to
   people in exchange for money or services.
   Outstanding shares may increase or decrease for other reasons too (such as stock splits
   and share buybacks, which we won’t get into here).
-  Later we discuss [several other subtleties](#counting-shares) about how shares are counted.
+  Later, we discuss [several subtleties](#counting-shares) in how shares are counted.
   - 🚧 What is a good overview on stock splits and share buyback.
     Key resources?
 - If you have stock, what ultimately matters is not the number of shares you hold, but the
   **percentage ownership** of the entire company that this number represents.
-  To determine the percentage of the company a certain number of shares represents, you
-  divide it by the number of outstanding shares.
+  To determine the percentage of the company a certain number of shares represents, divide
+  it by the number of outstanding shares.
   Even if you have a fixed number of shares, your percentage ownership will change over time
   as the outstanding shares change.
 
 ### Equity
 
 🄳 We say you have **equity** in a company when you have some kind of ownership or likely
-future ownership.
+future ownership of the company.
```

Comment:
I know! I was back and forth, I'm telling you! Ultimately "of" is the correct usage (think 'percentage of ownership,' 'portion of ownership,' those are the phrases that 'ownership in' is standing in for). Colloquially you would of course say "in." Kind of like with "vesting," nouns and verbs just have their own funny lives in this topic. My decision of "of" just in this spot was in the end to establish as clear as possible a definition so that all would grasp the concept before we drift into neologisms (altho who knows how old these finance type terms are). Will fix but now you know my thinking.

Reply:
Fixed.

Reply:
Sure. If you want my own logic, it would be that "kind of ownership" is more or less a "an ownership stake" *in* the company. If you read a news story it will talk about the owner *of* a football team, but Gates' stake *in* Microsoft.